rosidl_typesupport_connext_cpp/resource/srv__type_support.hpp.em

```diff
@@ -83,6 +83,11 @@ send_response__@(spec.srv_name)(
   void * untyped_replier,
   const rmw_request_id_t * request_header,
   const void * untyped_ros_response);
+
+ROSIDL_TYPESUPPORT_CONNEXT_CPP_PUBLIC
+const char *
+get_request_topic_name__@(spec.srv_name)(void * untyped_requester);
```

Comment:
This change does not appear in the parallel change is that expected? https://github.com/ros2/rmw_opensplice/pull/137/files